### ISSUE TITLE
a11y: List contains elements that should be outside of the list

### ISSUE THREAD

**Issue by iamjolly** [18F/all_sorns#134]
## Description:
There is a `div` with a `nav` element (for pagination) as a direct child of the parent unordered list in the search results page. This could be confusing and/or problematic for screen reader users. Per Accessibility Insights, "`<ul>` and `<ol>` must only directly contain `<li>`, `<script>` or `<template>` elements. [See more info here.](https://accessibilityinsights.io/info-examples/web/list/)"

## Screenshot:
![screenshot of list with problematic children](https://user-images.githubusercontent.com/1093423/105246735-e7c5f480-5b30-11eb-87d1-43f9bbbb3ab4.png)

## WCAG SC:
[Success Criterion 1.3.1 Info and Relationships (Level A)](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html): Information, structure, and relationships conveyed through presentation can be programmatically determined or are available in text.

Location of code:
[/app/views/sorns/search.html.erb, Line 168](https://github.com/18F/all_sorns/blob/41c4787de09eff4ff2c745250df070efa14504e2/app/views/sorns/search.html.erb#L168) 

## Recommended fix:
Move the following code to just after the closing `</ul>` element:
```
<div class="grid-offset-6 grid-col-6 margin-bottom-3">
  <%= paginate @sorns %>
</div>
```
